scripts/gha/deps_ios.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd $GITHUB_WORKSPACE
+
+git clone https://github.com/libsdl-org/SDL -b SDL2
```

Comment:
Better use `-b release-$SDL_VERSION`, so that we have same stable SDL2 version across all platforms.